assets/scripts/lobby/options.js

```diff
@@ -275,28 +275,45 @@ var userOptions = {
         }
     },
 
-   optionProposedTeamIcon: {
+   optionDisplayProposedTeamIcon: {
         defaultValue: "false",
         onLoad: function(){
-            if (docCookies.getItem("optionProposedTeamIcon") === "shield") {
-                $("#option_proposed_team_icon")[0].checked = true;
-                updateGunImage("shield");
+            //check if optionDisplayProposedTeamIcon exists in cookies
+            var isOptionExists = docCookies.hasItem("optionDisplayProposedTeamIcon");
+            var icon = "";
+            //if not, set it
+            if (isOptionExists === false) {
+                icon = 'gun';
+                //save it in cookie
+                docCookies.setItem("optionDisplayProposedTeamIcon", icon, Infinity);
+            } else {
+                //else load it
+                icon = docCookies.getItem("optionDisplayProposedTeamIcon")
+            }
+            //set check marks
+            if (icon === "shield") {
+                $("#option_display_proposed_team_icon")[0].checked = true;
             }
             else{
-                $("#option_proposed_team_icon")[0].checked = false;
-                updateGunImage("gun");
+                $("#option_display_proposed_team_icon")[0].checked = false;
             }
+            //update image on load
+            updateGunImage(icon);
         },
         initialiseEventListener: function(){
-            $("#option_proposed_team_icon")[0].addEventListener("click", function () {
+            $("#option_display_proposed_team_icon")[0].addEventListener("click", function () {
                 //when they press it...
-                var checked = $("#option_proposed_team_icon")[0].checked;
-                if(checked === true){
-                    updateGunImage("shield");
+                var isChecked = $("#option_display_proposed_team_icon")[0].checked;
+                var icon = "";
+                if(isChecked === true){
+                    icon = 'shield';
                 } else {
-                    updateGunImage("gun");
+                    icon = 'gun';
```

Comment:
Could you keep to the coding style of having `"<text>"` for strings and `'<char>'` for chars?

Reply:
accepted. will be resolved in the next commit.